src/app/app.component.ts

```diff
@@ -1,16 +1,19 @@
 import { Component } from '@angular/core';
-import { RouterOutlet } from '@angular/router';
+import { RouterLink, RouterLinkActive, RouterOutlet } from '@angular/router';
 import { TaskCreateComponent } from './task-create/task-create.component';
 import { TaskListComponent } from './task-list/task-list.component';
+import { HttpClientModule } from '@angular/common/http';
+
 
 @Component({
   selector: 'tasko-root',
   standalone: true,
-  imports: [RouterOutlet, TaskCreateComponent, TaskListComponent],
+  imports: [RouterOutlet, TaskCreateComponent, TaskListComponent, RouterLink, RouterLinkActive, HttpClientModule],
   templateUrl: './app.component.html',
   styleUrl: './app.component.scss'
 })
 export class AppComponent {
+noListNav: any;
```

Comment:
avoid using any type, create an interface/type for nav list, nav item or whatever model you need to use